lib/block_view_test.go

```diff
@@ -15521,7 +15521,7 @@ func TestNFTBasic(t *testing.T) {
 	{
 		// Balance before.
 		m0BalBeforeNFT := _getBalance(t, chain, nil, m0Pub)
-		require.Equal(uint64(27), m0BalBeforeNFT)
+		require.Equal(uint64(28), m0BalBeforeNFT)
```

Comment:
@lazynina, why did all these balance values change? I can't figure it out...

Reply:
See this comment: https://github.com/deso-protocol/core/pull/120#discussion_r714284576

I added a new attribute to the body schema and was going to only add omitempty for the new VideoURLs attribute to maintain backwards compatibility. However, if we add omitempty to the Body and ImageURLs as well, we can save  a KB per post

Reply:
Oooooh, of course! I was misreading this change.  I thought the posts were costing *more* but they are actually costing *less*, which makes sense.  Awesome.